pynecone/utils/build.py

```diff
@@ -126,6 +126,12 @@ def setup_frontend(root: Path):
     Args:
         root: root path of the project.
     """
+    # Initialize the web directory if it doesn't exist.
+    web_dir = prerequisites.create_web_directory(root)
```

Comment:
When use call pc init, the web folder will be there.
Because pc init will do 
```
        prerequisites.install_bun()
        prerequisites.initialize_web_directory()
```

Why we still need to use this?
`web_dir = prerequisites.create_web_directory(root)`